cookbook/deployment/cluster/auth_setup.py

```diff
@@ -1,8 +1,498 @@
 """
+Authentication in Flyte
+-----------------------
+
+Flyte ships with a canonical implementation of OpenIDConnect client and OAuth2 Server, integrating seamlessly into an organization's existing identity provider.
+
+This sesction includes:
+- Overview
+- Authentication Setup
+- Migrating Your Authentication Config
+
```

Comment:
Is there a way to add an anchor links here? With links to the individual section headings below?

Reply:
added reference links

Reply:
Thank you Niels!